src/main/java/lotto/vo/LottoResult.java

```diff
@@ -0,0 +1,46 @@
+package lotto.vo;
+
+import java.util.Map;
+
+public class LottoResult {
```

Comment:
일급 컬렉션 👍 복수 명사를 클래스 이름으로 사용하면 컬렉션임을 더 쉽게 표현할 수 있겠죠?

```suggestion
public class LottoResults {
```